### DISCUSSION TITLE
Anzeige "Verbrauch" bis 150%

### DISCUSSION THREAD

**Discussion by WolfImBusch** [solectrus/solectrus#3299] [Q&A]
Hallo,

ich bin etwas verwirrt über die Anzeige ... :-) Bei "Aktuell" und "Tag" wählt Solectrus eine (vermutlich automatische) Skalierung, die einen Verbrauch bis zu 150% darstellen könnte. Ursache (habe ich nicht geprüft weil ich zu doof bin) könnte sein, dass einer meiner über MQTT eingesammelten Werte über 100% ist. Würde es Sinn machen, die Skalierung fix auf 100%  einzustellen?

Danke und viele Grüße!!!

![Skalierung Verbrauch](https://github.com/user-attachments/assets/b32dd6ad-e867-4fc1-bc6b-02b56cd0790f)

**Comment by ledermann** [comment]
Gut erkannt, das liegt an der automatischen Skalierung, der ich wohl etwas nachhelfen muss. 

Issue #3301 ist angelegt.
